src/main/resources/examples/resource/initech.tps.report.v1.json

```diff
@@ -100,5 +100,12 @@
             ]
         }
     },
+    "serviceLink": {
```

Comment:
Since a service will have several resource links, should we rename this `resourceLink`?

Reply:
think this naming came about because it's a link to a service that isn't CloudFormation. each schema is one resource type, so i'm not too worried here, besides the examples are hopefully obvious.

Reply:
changed my mind on this, think you're right and resource link is more natural. fixed!